common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -24,7 +24,7 @@ private UserRequestValidator() {}
    *
    * @param userRequest Request
    */
-  public static void validateCreateUser(Request userRequest) {
+  public static void validateCreateUserRequest(Request userRequest) {
```

Comment:
This PR has several files which are not related (e.g. unit test files). Can you check and revert once done?